tests/dash/configs/privatelink_config.py

```diff
@@ -74,6 +79,20 @@
     }
 }
 
+ENI_TRUSTED_VNI_CONFIG = {
+    f"DASH_ENI_TRUSTED_VNI_TABLE:{ENI_ID2}": {
```

Comment:
No table with this name, is this an ENI 
```suggestion
    f"DASH_ENI_TABLE:{ENI_ID2}": {
```